### PR TITLE
updated jms serializer to 1.0 stable

### PR DESCRIPTION
**Pull request by OskarStark**

### CODE REVIEW

**Comment by OskarStark** [comment]
refs #522

**Comment by OskarStark** [comment]
@Soullivaneuh @rande looks like this error is not PR related, can you please have a look?

bildschirmfoto 2015-09-03 um 09 15 28

**Comment by Soullivaneuh** [comment]
Travis job restarted.

File: composer.json [hunk 1]
```diff
@@ -29,7 +29,7 @@
         "sonata-project/datagrid-bundle": "~2.2@dev",
 
         "friendsofsymfony/rest-bundle": "~1.1",
-        "jms/serializer-bundle": "~0.11",
+        "jms/serializer-bundle": "~1.0",
```
Comment (Soullivaneuh) [inline composer.json]: `~0.11|~1.0` instead.

**Comment by OskarStark** [comment]
updated @Soullivaneuh

**Comment by OskarStark** [comment]
done @Soullivaneuh 

works on my maschine:
bildschirmfoto 2015-09-03 um 10 08 29

**Comment by Soullivaneuh** [comment]
Yes but Travis need to be fixed first.

Will take a look.

**Comment by OskarStark** [comment]
Thank you!

**Comment by OskarStark** [comment]
@Soullivaneuh any news?

**Comment by OskarStark** [comment]
can i help you with this travis problem?

**Comment by Soullivaneuh** [comment]
I just have to backport configuration from master.

Just need time, be patient. ;-)

**Comment by Soullivaneuh** [comment]
@OskarStark #524 solved Travis issue.

Please rebase your PR.

**Comment by OskarStark** [comment]
@Soullivaneuh i will, if this button mentioned in the blogpost to update the base branch does not work yet.
can you please check this?

https://github.com/blog/2051-protected-branches-and-required-status-checks

**Comment by OskarStark** [comment]
hmm looks, like something went wrong with the rebase :(

**Pull request closed without merging**